pom.xml

```diff
@@ -24,7 +24,7 @@
             <groupId>javax.servlet</groupId>
             <artifactId>javax.servlet-api</artifactId>
             <version>4.0.1</version>
-            <scope>provided</scope>
+
```

Comment:
why have you removed this line?

Reply:
The last checklist gave information about work without main.

Reply:
you should not remove it if you are testing your app with the browser